project.yaml

```diff
@@ -1,4 +1,14 @@
 targets:
+  noble:
```

Comment:
I know this is auto-generated, but can we put this after `mantic`?

Reply:
Done. I don't think it really interferes to change the order for a followup auto-generation since it was set as first entry this time.

Reply:
Right, it just uses the tag comment and ignores the rest of the section.